code/game/machinery/computer/tram_controls.dm

```diff
@@ -4,7 +4,7 @@
 	icon_screen = "tram"
 	icon_keyboard = "atmos_key"
 	circuit = /obj/item/circuitboard/computer/tram_controls
-	flags_1 = NODECONSTRUCT_1
+	flags_1 = NODECONSTRUCT_1 | SUPERMATTER_IGNORES_1 //What fun is it if we get consumed by the SM and can't move the tram anymore?
```

Comment:
kinda dissapointed in this

Reply:
I mean, it's already indestructible by other flags. This is only to make it so it doesn't dust when the supermatter rams into it